data.sql

```diff
@@ -52,3 +52,134 @@ SET owner_id = (
         WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
     END
 );
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your Key Features section more descriptive by adding more exciting features in this section to create excitement in the viewers of your app 👍